### PR TITLE
fix: module assets is in buildInfo

### PR DESCRIPTION
**Pull request by clarkdo**
**What kind of change does this PR introduce?**
bugfix

**Did you add tests for your changes?**

**Summary**

`module.assets` should be `module.buildInfo.assets` due to https://github.com/webpack/webpack/pull/6111

**Does this PR introduce a breaking change?**

No

**Other information**

### CODE REVIEW

**Comment by ooflorent** [comment]
Could you add some tests?

**Comment by clarkdo** [comment]
@ooflorent I will have a try 😄

File: test/statsCases/modules-assets/expected.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+155613d841c408f2a34b9c81cdfd3e8c.png   37.7 KiB          [emitted]  
+                                0.js  607 bytes       0  [emitted]  
+                             main.js   6.76 KiB       1  [emitted]  main
+Entrypoint main = main.js
+chunk    {0} 0.js 99 bytes <{1}> [rendered]
+    [0] ./node_modules/a/1.png 82 bytes {0} [built]
+    [1] ./node_modules/a/index.js 17 bytes {0} [built]
+chunk    {1} main.js (main) 12 bytes >{0}< [entry] [rendered]
+    [2] ./index.js 12 bytes {1} [built]
+   [0] ./node_modules/a/1.png 82 bytes {0} [built]
```
Comment (sokra) [inline test/statsCases/modules-assets/expected.txt]: The `assets` info isn't in the stats string output. It's only in the stats json.

Maybe you can expand this PR a bit and add support for showing emitted files in the Stats.

I could image a small flag like this:

``` text
[0] ./node_modules/a/1.png 82 bytes {0} [built] [1 asset]
```

You also need include a flag `moduleAssets` to show/hide this information. It should be enabled by default in json output but disabled by default for toString, so it's backward-compatible.

Comment (clarkdo) [reply test/statsCases/modules-assets/expected.txt]: @sokra 
Sure, I will try to work on int.

**Comment by clarkdo** [comment]
@sokra Please review the my changes 😸.

File: lib/Stats.js [hunk 2]
```diff
@@ -867,6 +873,9 @@ class Stats {
 			if (module.built) {
 				colors.green(" [built]");
 			}
+			if (module.assets && module.assets.length) {
+				colors.magenta(` [${module.assets.length} asset]`);
```
Comment (sokra) [inline lib/Stats.js]: for `module.assets.length > 1` it should be asset**s** instead of asset

Comment (clarkdo) [reply lib/Stats.js]: @sokra 
Thank you for the review, I have changed it.

I thinks it's better to extract `module.assets.length` into a variable such as `const assetsSize = module.assets.lengt`, but for keeping the same behavior with other sources(`module.warnings, module.errors`), I directly used `module.assets.length`.

**Comment by clarkdo** [comment]
@sokra 
The building error should not be related to this pr.

And about the PR Quality Review, codes have been prettied by `pretty-files` and checked by eslint, if I change as Codacy showed, it will be failed in eslint.

**Comment by sokra** [comment]
Thanks

**Pull request merged**